server/tools/video_tools.py

```
@@ -0,0 +1,252 @@
+import random
+import time
+import traceback
+import os
+import json
+from typing import Optional, Annotated
```

Flake8 (7.2.0)

[error] 'typing.Optional' imported but unused

(F401)